### PR TITLE
HyperShift-Agent: add assisted-service

### PR DESCRIPTION
**Pull request by LiangquanLi930**
Due to the changes in this PR: https://github.com/openshift/release/pull/36470/files, we need to add `assisted-service`

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.13-periodics-e2e-agent-ovn-conformance

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.13-periodics-e2e-agent-ovn-conformance

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.13-periodics-e2e-agent-ovn-conformance

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.12__periodics.yaml [hunk 1]
```diff
@@ -15,6 +15,10 @@ base_images:
     name: ocm-2.7
     namespace: edge-infrastructure
     tag: assisted-installer-controller
+  assisted-service:
```
Comment (eranco74) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.12__periodics.yaml:18]: I think we can drop 4.12.
There's no need to test it

Comment (LiangquanLi930) [reply ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.12__periodics.yaml]: done

**Review by CrystalChun — APPROVED** [review]
/lgtm

**Pull request merged**